cmd/neofs-node/config/contracts/config.go

```diff
@@ -0,0 +1,67 @@
+package contractsconfig
+
+import (
+	"fmt"
+
+	"github.com/nspcc-dev/neo-go/pkg/util"
+	"github.com/nspcc-dev/neofs-node/cmd/neofs-node/config"
+)
+
+const (
+	subsection = "contracts"
+)
+
+// Netmap returns value of "netmap" config parameter
+// from "contracts" section.
+//
+// Throws panic if value is not set.
```

Comment:
```suggestion
// Throws panic if value is not a LE hex-encoded string.
```
and for the other methods.

Reply:
Fixed